src/sick_tim_common_usb.cpp

```diff
@@ -265,6 +265,7 @@ int SickTimCommonUsb::sendSOPASCommand(const char* request, std::vector<unsigned
   if (device_handle_ == NULL) {
     ROS_ERROR("LIBUSB - device not open");
     diagnostics_.broadcast(diagnostic_msgs::DiagnosticStatus::ERROR, "LIBUSB - device not open.");
+    sleep(1);
```

Comment:
@jspricke @mintar 

I'll find an acceptable place for this.  I think it should go into the common_usb as the tcp/ip version has an intrinsic timeout/wait inside of it.  I'm not against moving it to the outer loop, but it would add two distinct delays to the tcp/ip version with only one being used in the usb version.

Thoughts?